src/TransWithoutContext.js

```diff
@@ -13,7 +13,7 @@ function hasChildren(node, checkLength) {
 
 function getChildren(node) {
   if (!node) return [];
-  return node.props ? node.props.children : node.children;
+  return getAsArray(node.props ? node.props.children : node.children);
```

Comment:
I would not always return an array... but maybe only if i18nIsDynamicList

Reply:
Or assert you always pass an array when using `i18nIsDynamicList` (as the name suggests it's about lists and not single elements) -> code has to look like:

```js
   <Trans>
      <React.Fragment i18nIsDynamicList>
        {list.length === 1 ? [<div>{list[0]}</div>] : list.map((x) => <div> {x} </div>)}
      </React.Fragment>
    </Trans>
```

Reply:
Thinking about this more, the following would produce the exact same result:
```ts
    <Trans>
      {list.length === 1 ? list[0] : list.map((x) => <div>{x}</div>)}
    </Trans>
```

Am I correct in the above?

Either way, I think allowing single elements would make this prop easier to use. I don't quite see any reasoning why disallowing single elements would make sense, other than the fact that the prop mentions the word "list" (which I think would be better named `i18nHasDynamicContent` or something similar).

Let me know the final decision though, and I can change/close this PR accordingly.

Reply:
for sure I would not always return an array, but if, then only for i18nIsDynamicList